src/model/bioprotocol.v1.yaml

```diff
@@ -0,0 +1,25 @@
+$schema: http://json-schema.org/draft-04/schema#
+title: BioProtocol
+type: array
+items:
+    type: object
+    properties:
+        protocol_sequencing_number:
+            description: "section ID, looks like 's4-1'"
+            type: string
+        protocol_title:
+            description: "section title, looks like 'Isolation of LDs and treatments'"
+            type: string
+        is_protocol:
+            type: boolean
+        protocol_status:
+            description: "numeric code corresponding to the current status of a protocol"
+            type: "integer"
+            minimum: 0
```

Comment:
anything better than a number? Is there a translation table from `0`, `1` to keywords?

Reply:
there isn't

Reply:
I've asked Dong to elaborate

Reply:
Dong has gotten back to us. They actually switched from a boolean *to* an integer but his reply doesn't read like they intend to expand the number of different states any time soon.

I can store the integer and return a boolean value instead. When/if BP decide to add more statuses we can tweak things then.